NotesBackEnd/src/Notes.Api.Services/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
You should not be needing this file here. Only the `app.config` (or `web.config`) of an executable assembly is used by the framework (for example `tests` are executable as well as the web app)